pkg/scanners/scannerv4/scannerv4.go

```diff
@@ -366,3 +406,26 @@ func newNodeScanner(integration *storage.NodeIntegration) (*scannerv4, error) {
 
 	return scanner, nil
 }
+
+// NewVirtualMachineScanner provides a scannerv4 instance that is able to scan virtual machines
+func NewVirtualMachineScanner() (types.VirtualMachineScanner, error) {
+	matcherEndpoint := DefaultMatcherEndpoint
+
+	ctx := context.Background()
+	scannerClient, err := client.NewGRPCScanner(
+		ctx,
+		client.WithMatcherAddress(matcherEndpoint),
```

Comment:
Apologies missed this before, the matcher endpoint must be pulled from the config when available. Some users have had to customize it in the past.

example:
https://github.com/stackrox/stackrox/blob/915d5d7a39c4f2d1d9839746bdc2a084bbcab83d/pkg/scanners/scannerv4/scannerv4.go#L87-L90

Reply:
I intended to have the configurability of the virtual machine scanner addressed with ROX-31177
I'll open a follow-up PR for the scanner configuration part